blog/_posts/2019-09-05-put-down-particle-accelerator.md

```diff
@@ -0,0 +1,78 @@
+
+Make's superpowers came from changing the format: a build.sh script has to encode all the logic upfront, but a declarative Makefile lets the build tool engine introspect to provide a fast and consistent experience. MDEs will need a declarative description of how to build and deploy your microservice app. Because distributed systems involve the state of running services, you'll need a richer language to describe dependencies beyond the file-level data that build tools encode.
+
+This description of your app's molecule will include how to build each component microservice's atom. Crafting a great experience at the molecular level won't leave much time to work on lower levels. The same way MDEs shouldn't write their own network stacks, a good MDE will let you use whatever build tool you already use.
+
+MDEs will be reactive to provide the best of both IDEs and Kubernetes. Like IDEs, MDEs will watch your filesystem to upate when you save. Like Kubernetes they'll watch your cluster to respond to unexpected events. The same way Kubernetes frees you from restarting a process, MDEs free you from remembering to do a build or refresh a log.
+
+## Early MDEs
+One popular declarative description of how to build and deploy your microservice app is Docker Compose. [Docker Compose](https://docs.docker.com/compose/) is a proto-MDE: it can build and start, but won't react as you edit with incremental updates. To be clear: we love Docker Compose! It's incredibly popular, a testament to the value it delivers, and users can hack in updates via mount points. Other proto-MDEs work outside containers, like foreman, the cleverly-named go-implementetation goreman, and modd offer similar functionality outside containers.
+
+I work building [Tilt](https://tilt.dev), a Kubernetes-focused MDE. If you use Kubernetes in production, you need Tilt in development. There are other MDE projects, like [Skaffold](https://skaffold.dev), [Tekton](https://tekton.dev) (sort of an MDE), [Garden](https://garden.io), [Okteto](https://okteto.com), or [Kelda](https://kelda.io).
+
+I'm especially proud of a few Tilt features that really exemplify what MDEs can be, how they can offer a better experience, and how Tilt's going to grow:
+* The [Tiltfile](https://docs.tilt.dev/tutorial.html) is an ergonomic way to define how to assemble the molecule of your app, from the simple (our [tutorial](https://docs.tilt.dev/tutorial.html) highlights getting started with just Dockerfiles and Kubernetes YAML) or complex (using [`local`](https://docs.tilt.dev/tiltfile_concepts.html) to call an existing yaml generator).
```

Comment:
> using local to call an existing yaml generator

imo actually bringing a bit of our syntax into it makes it more about syntax and distracts from the point of the sentence. maybe consider:

> ...or complex (calling out to an existing YAML generator, like Helm, or your own custom one)

Reply:
I rephrased a bit, but still wanted a concrete example and link.